cellprofiler/gui/figure.py

```diff
@@ -654,12 +654,14 @@ def in_bounds(image, xi, yi):
         return not (image is None or xi >= image.shape[1] or yi >= image.shape[0] or xi < 0 or yi < 0)
 
     def on_mouse_move_measure_length(self, event, x0, y0, x1, y1):
-        if event.xdata is None or event.ydata is None:
-            return
-        xi = int(event.xdata + .5)
-        yi = int(event.ydata + .5)
-        fields = self.get_fields(event, yi, xi, x1)
-
+        # Get the fields later used in the statusbar
+        fields = [""]
```

Comment:
You should consider using `fields = None`.

Reply:
updated; had to change the part that updates the statusbar as well